content/english/facts/soy-is-bad-for-you.md

```diff
@@ -0,0 +1,191 @@
+
+This talks specifically about beneficial effects (which has been the focus of a lot of research due to health claims for tofu in the 1990s), the full scope of large dosages of phytoestrogens (whatever may constitute large dosages!) can have negative effects.
+
+> There are also real concerns that excessive amounts of isoflavones in serum may promote other hormone-related problems.  
+> - [An updated review of dietary isoflavones: Nutrition, processing, bioavailability and impacts on human health - Critical Reviews in Food Scient and Nutrition - 2017](http://web.archive.org/web/20191015205851/https://www.researchgate.net/profile/Humayoun_Akhtar/publication/283791348_An_Updated_Review_of_Dietary_Isoflavones_Nutrition_Processing_Bioavailability_and_Impacts_on_Human_Health/links/5a2f732a4585155b617a45fd/An-Updated-Review-of-Dietary-Isoflavones-Nutrition-Processing-Bioavailability-and-Impacts-on-Human-Health.pdf?origin=publication_detail)
+
+and 
+
+> [P]hytoestrogens are also considered endocrine disruptors, indicating that they have the potential to cause adverse health effects such as infertility and increased risks of cancer in oestrogen-sensitive organs.  
+> - [The potential health effects of dietary phytoestrogens - British Journal of Pharmacology - 2017](https://web.archive.org/web/20190922065924/https://bpspubs.onlinelibrary.wiley.com/doi/full/10.1111/bph.13622)  
+
+That said, the few studies that show negative effects have major drawbacks such as:
+
+1. Being performed on animals, which apart from the ethical concerns do not even necessarily yield valid results for humans! 
+2. Being performed in-vitro (i.e. in a petri-dish on small clusters of cells)
+3. Drawing conclusions from statistical (e.g. epidiemological) data which risks confusing mere correlation with cause.
+
+
+>  These adverse effects have been mainly suggested based on data from in vitro, animal or epidemiological studies. Clinical studies often report the absence of adverse effects. 
+> - [The potential health effects of dietary phytoestrogens - British Journal of Pharmacology - 2017](https://web.archive.org/web/20190922065924/https://bpspubs.onlinelibrary.wiley.com/doi/full/10.1111/bph.13622)  
+
+and
+
+> Concerns about the adverse effect of soy consumption were originally based on animal data.  
+> - [Soy, Soy Foods and Their Role in Vegetarian Diets - Nutrients - 2018](https://web.archive.org/web/20191002195620/https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5793271/pdf/nutrients-10-00043.pdf/)
+
+
+## 3. It's unlikely you'll ingest enough to have adverse effects, but vary your protein intake anyway!
+
+Having a varied diet is rarely bad advice. And while it seems like you are unlikely to eat enough to adversely affect your health, it's also pretty much impossible to measure how much phytoestrogen you're ingesting on a daily basis without specialised equipment. And even if you could, there are no official recommended daily allowances to stay below.
+
```

Comment:
Could you say varies plant-based diet? Sounds a little too much like a nod towards meat as part of a varied diet

Reply:
Oh god I didn't even think of that! Well spotted.